index.js

```diff
@@ -1,26 +1,37 @@
-'use strict';
+"use strict";
 
 /**
  * This function receives a string input and converts it to mOcKiNgCaSe.
  * @param {(string | string[])} input - string or array of strings to be converted
  * @param {object} [options={random: false}] - options for converting
```

Comment:
can you please add `onlyLetters` also here
```suggestion
 * @param {object} [options={random: false, onlyLetters: false}] - options for converting
```